spark-bigquery-dsv1/spark-bigquery-dsv1-spark3-support/pom.xml

```diff
@@ -19,10 +19,15 @@
     </license>
   </licenses>
   <dependencies>
+    <dependency>
+      <groupId>${project.groupId}</groupId>
+      <artifactId>spark-bigquery-connector-common</artifactId>
+      <version>${project.version}</version>
+    </dependency>
     <dependency>
       <groupId>org.apache.spark</groupId>
       <artifactId>spark-sql_2.12</artifactId>
-      <version>3.0.0</version>
+      <version>3.2.0</version>
```

Comment:
Should this be 3.3.0?

Reply:
I prefer to use the lowest spark version that I can